Error when extracting data EID

## Description
Whenever I play a song it never actually plays the song. I am very new to Lavalink so I don't exactly know how to interpret the log messages but what I think is important is:
"no PLAYER_JS_URL in youtube root EID"
"Error when extracting data EID"
"com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track"
"Caused by: java.lang.RuntimeException: Error when extracting data EID"

I don't know exactly if I am the problem or if something else is at fault so I am very sorry if I bother anyone with this but I am sitting at this issue for 5 hours now and don't know where else to find more information about this.


## Version info
**Client used:** WaveLink 0.9.9

Output of `java -version`:
```
$ java -version
openjdk version "13" 2019-09-17
OpenJDK Runtime Environment (build 13+33)
OpenJDK 64-Bit Server VM (build 13+33, mixed mode, sharing)
```

<!-- You must include your lavalink logs -->
<!-- If you can't provide a log, at least provide the output of "java -jar Lavalink.jar --version" -->
I shortened the log quite a bit because there was a huge dump of to me unnecessary code in the middle. I can give that to you if need be but for visibility I'll leave it out
`2021-03-31 22:56:10.981  INFO 9656 --- [main] lavalink.server.Launcher                 : 

[32m       .  [31m _                  _ _       _    [32m__ _ _
[32m      /\\ [31m| | __ ___   ____ _| (_)_ __ | | __[32m\ \ \ \
[32m     ( ( )[31m| |/ _` \ \ / / _` | | | '_ \| |/ /[32m \ \ \ \
[32m      \\/ [31m| | (_| |\ V / (_| | | | | | |   < [32m  ) ) ) )
[32m       '  [31m|_|\__,_| \_/ \__,_|_|_|_| |_|_|\_\[32m / / / /
[0m    =========================================[32m/_/_/_/[0m

	Version:        8be3fec7084caa2659e92c28f1f43070fcb84dcd-SNAPSHOT
	Build:          1209
	Build time:     17.02.2021 23:24:14 UTC
	Branch          master
	Commit:         8be3fec
	Commit time:    17.02.2021 23:18:16 UTC
	JVM:            13
	Lavaplayer      1.3.65

2021-03-31 22:56:11.038  INFO 9656 --- [main] lavalink.server.Launcher                 : Starting Launcher on xxx with PID xxx (xxx\Lavalink.jar started by xxx in xxx)
2021-03-31 22:56:11.038  INFO 9656 --- [main] lavalink.server.Launcher                 : No active profile set, falling back to default profiles: default
2021-03-31 22:56:11.093  WARN 9656 --- [background-preinit] o.s.h.c.j.Jackson2ObjectMapperBuilder    : For Jackson Kotlin classes support please add "com.fasterxml.jackson.module:jackson-module-kotlin" to the classpath
2021-03-31 22:56:11.884  WARN 9656 --- [main] io.undertow.websockets.jsr               : UT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used
2021-03-31 22:56:11.899  INFO 9656 --- [main] io.undertow.servlet                      : Initializing Spring embedded WebApplicationContext
2021-03-31 22:56:11.899  INFO 9656 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 834 ms
2021-03-31 22:56:12.091  WARN 9656 --- [main] l.server.config.SentryConfiguration      : Turning off sentry
2021-03-31 22:56:12.237  INFO 9656 --- [main] c.s.d.l.tools.GarbageCollectionMonitor   : GC monitoring enabled, reporting results every 2 minutes.
2021-03-31 22:56:12.530  INFO 9656 --- [main] lavalink.server.config.KoeConfiguration  : OS: Windows 10, Arch: amd64
2021-03-31 22:56:12.530  WARN 9656 --- [main] lavalink.server.config.KoeConfiguration  : This system and architecture appears to not support native audio sending! GC pauses may cause your bot to stutter during playback.
2021-03-31 22:56:12.649  INFO 9656 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2021-03-31 22:56:12.795  INFO 9656 --- [main] org.xnio                                 : XNIO version 3.3.8.Final
2021-03-31 22:56:12.802  INFO 9656 --- [main] org.xnio.nio                             : XNIO NIO Implementation Version 3.3.8.Final
2021-03-31 22:56:12.840  INFO 9656 --- [main] o.s.b.w.e.u.UndertowServletWebServer     : Undertow started on port(s) 2333 (http) with context path ''
2021-03-31 22:56:12.842  INFO 9656 --- [main] lavalink.server.Launcher                 : Started Launcher in 2.128 seconds (JVM running for 2.397)
2021-03-31 22:56:12.844  INFO 9656 --- [main] lavalink.server.Launcher                 : You can safely ignore the big red warning about illegal reflection. See https://github.com/Frederikam/Lavalink/issues/295
2021-03-31 22:56:19.185  INFO 9656 --- [XNIO-1 task-1] io.undertow.servlet                      : Initializing Spring DispatcherServlet 'dispatcherServlet'
2021-03-31 22:56:19.186  INFO 9656 --- [XNIO-1 task-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2021-03-31 22:56:19.191  INFO 9656 --- [XNIO-1 task-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 5 ms
2021-03-31 22:56:19.208  INFO 9656 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Incoming connection from /127.0.0.1:50488
2021-03-31 22:56:19.242  INFO 9656 --- [XNIO-1 task-1] lavalink.server.io.SocketServer          : Connection successfully established from /127.0.0.1:50488
2021-03-31 22:56:26.300  INFO 9656 --- [XNIO-1 task-2] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "ytsearch:Dandelion Galantis"
2021-03-31 22:56:27.121  INFO 9656 --- [lava-daemon-pool-info-loader-2-thread-1] lavalink.server.player.AudioLoader       : Loaded playlist Search results for: Dandelion Galantis
2021-03-31 22:56:27.508  INFO 9656 --- [XNIO-1 I/O-4] lavalink.server.io.SocketServer          : {"op": "play", "guildId": "708452533856894977", "track": "QAAAlAIAMkdhbGFudGlzICYgSlZLRSAtIERhbmRlbGlvbiBbT2ZmaWNpYWwgTHlyaWMgVmlkZW9dAAhHYWxhbnRpcwAAAAAAAiqwAAtMMFk3R2lTbDF6OAABACtodHRwczovL3d3dy55b3V0dWJlLmNvbS93YXRjaD92PUwwWTdHaVNsMXo4AAd5b3V0dWJlAAAAAAAAAAA=", "noReplace": false, "startTime": "0"}
2021-03-31 22:56:27.821  INFO 9656 --- [XNIO-1 I/O-4] lavalink.server.io.SocketServer          : {"op": "voiceUpdate", "guildId": "708452533856894977", "sessionId": "ac2be58495dac0ea8cabda078c433de3", "event": {"token": "5ff07e82736821b9", "guild_id": "708452533856894977", "endpoint": "eu-central1112.discord.media:443"}}
2021-03-31 22:56:28.318  WARN 9656 --- [lava-daemon-pool-playback-1-thread-1] c.s.d.lavaplayer.tools.ExceptionTools    : no PLAYER_JS_URL in youtube root EID: 4ad2e966-3b5a-45b8-bd6f-3bb01931e921, html: <!doctype html><html lang="en" dir="ltr"><head><base href="https://consent.youtube.com/"><meta name="referrer" content="origin"><link rel="canonical" href="https://consent.youtube.com/m"><meta name="viewport" content="initial-scale=1,minimum-scale=1,maximum-scale=5,width=device-width"><link rel="shortcut icon" href="//www.google.com/favicon.ico"><script data-id="_gd" nonce="3+hR+xEx3VRcrhT86uuamA">window.WIZ_global_data = {"DndLYb":"","DpimGf":false,"EP1ykd":["/_/*"],"FdrFJe":"-3528863320384077195","Im6cmf":"/_/ConsentUi","LVIXXb":1,"LoQv7e":false,"MT7f9b":[],"Mypbod":"https://www.googleapis.com/reauth","QrtxK":"","R6pIad":"%.@.]\n","S06Grb":"","S1NZmd":false,"Yllh3e":"%.@.1617224187602832,173895733,218196872]\n","cfb2h":"boq_identityfrontenduiserver_20210328.17_p0","eNnkwf":"","eptZe":"/_/ConsentUi/","fPDxwd":[1763433,1772879],"gGcLoe":false,"nQyAE":{"vEMF5e":"false","TYERre":"false","GSWfhb":"true","EoymAc":"false","FbHgvb":"false","P1ceCf":"false","wcLcde":"false","tBSlob":"false","XqMd3":"false","wNIRyf":"false","G25Msb":"false","A71CTb":"false","o10Fod":"false","APYQvd":"false","Dpi3Gf":"false","lttELb":"false","xIaf0e":"false","UcYwHb":"false","kQrwQd":"false","ihOA2":"false","Ozjmee":"false"},"qwAQke":"ConsentUi","qymVe":"","rtQCxc":-120,"thykhd":"AKH95eu-Dbm3Fq411tzykDrAHY-nsZoY2j4FkQ6Iom0r0D1YMl78twXl7oB2jfZWRpEI9k_Pa4ocBDCItJduE4UpUtbD7--JOg854ThoHL983wIXTegLmXU6xOc\u003d","vAyiz":"ChYIhJ/Vl9SqqNaHARCY2aa56o+wqaYB","w2btAe":"%.@.null,null,\"\",false,null,null,true,false]\n","zChJod":"%.@.]\n"};</script><script nonce="3+hR+xEx3VRcrhT86uuamA">(function(){/*
`
huge dump was here
`
2021-03-31 22:56:28.361  WARN 9656 --- [lava-daemon-pool-playback-1-thread-1] c.s.d.lavaplayer.tools.ExceptionTools    : Error when extracting data EID: bd6fe68f-a055-4742-993e-a93e0d21683b, mainJson: [there was stuff here but i removed it]
2021-03-31 22:56:28.383 ERROR 9656 --- [lava-daemon-pool-playback-1-thread-1] c.s.d.l.t.p.LocalAudioTrackExecutor      : Error in playback of L0Y7GiSl1z8

com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:117) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349) ~[lavaplayer-1.3.65.jar!/:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:830) ~[na:na]
	Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.3.65
  os.arch: amd64
  os.name: Windows 10
  os.version: 10.0
  java.vendor: Oracle Corporation
  java.version: 13
  java.runtime.version: 13+33
  java.vm.version: 13+33
Caused by: java.lang.RuntimeException: Error when extracting data EID: bd6fe68f-a055-4742-993e-a93e0d21683b, mainJson<redacted>
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.throwWithDebugInfo(ExceptionTools.java:135) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.load(DefaultYoutubeTrackDetailsLoader.java:70) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.loadDetails(DefaultYoutubeTrackDetailsLoader.java:45) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:76) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:42) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:105) ~[lavaplayer-1.3.65.jar!/:na]
	... 4 common frames omitted
Caused by: java.lang.RuntimeException: no PLAYER_JS_URL in youtube root EID: 4ad2e966-3b5a-45b8-bd6f-3bb01931e921, html<redacted>
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.throwWithDebugInfo(ExceptionTools.java:135) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.augmentWithPlayerScript(DefaultYoutubeTrackDetailsLoader.java:254) ~[lavaplayer-1.3.65.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.load(DefaultYoutubeTrackDetailsLoader.java:65) ~[lavaplayer-1.3.65.jar!/:na]
	... 8 common frames omitted

`

--- a/build.gradle
+++ b/build.gradle
@@ -54,7 +54,7 @@ subprojects {
     ext {
         //@formatter:off
         
-        lavaplayerVersion               = '1.3.73'
+        lavaplayerVersion               = '1.3.74'
         lavaplayerIpRotatorVersion      = '0.2.3'
         jdaNasVersion                   = '1.1.0'
         jappVersion                     = '1.3.2-MINN'

